pilot/cmd/istioctl/controlplane/yaml.go

```diff
@@ -0,0 +1,64 @@
+// Copyright 2017 Istio Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package controlplane
+
+import (
+	"bytes"
+	"fmt"
+	"strings"
+
+	"k8s.io/helm/pkg/chartutil"
+	"k8s.io/helm/pkg/engine"
+	"k8s.io/helm/pkg/proto/hapi/chart"
+	"k8s.io/helm/pkg/timeconv"
+)
+
+func yamlFromInstallation(values, namespace, helmChartDirectory string) (string, error) {
```

Comment:
Please add unit tests for the new functions.

Reply:
Also we need some tests to ensure that the yamls can be generated form helm charts correctly.

Otherwise, any future change to the helm files could break this command.

Reply:
PTAL. No tests for this part specifically as we need to fix up helm templates first, but tests for the other parts.